Feat 2135 pagelayouts content (#2186)
grommet/hpe-design-system: aries-site/src/pages/templates/app-layouts.mdx

  <AppAnatomy />
</Example>

1. **App Container**
- The container serves as the outer shell of a page UI.
- This is the parent component containing all HTML elements within a grid.

> - It's responsibility is to organize each of the page components, how they 

Reviewer: ```suggestion
- Its responsibility is to organize each of the page components, how they 
```

relate to eachother, the spacing between each, and how they respond at 
various screen widths.

2. **Global Header & Footer**  
- The Global Header, also known as the [HPE Common Header and Footer 
Web Service](/templates/global-header), provides a tunable, yet consistent 
header and footer to be used across all HPE applications.
- No matter the screen size, the Global Header & Footer will always be full 
width, occupying as much space as available.

3. **Page Container**  
- The Page Container contains the Page Content and the Page Header.
- It is responsible for the layout for the main content of a page. 
- Page container width may be [Wide](#wide-default), [Narrow](#narrow), of [Full](#full).

> 🏗️ **Coming soon:**

Reviewer: I found this icon a little bit different to see / interpret given the size -- could we do something like construction cone or construction sign? something with a little less detail

Author: @bahriaditi

  <PageContainerWide />
</Example>

**Use cases include:**
- Dashboards 
- Tables 
- Lists
- Charts

#### Narrow
- The "narrow" option scales the content width until a maximum width of `768px`. 

> - If the screen is larger than 768, the Page Container will be center aligned 

Reviewer: ```suggestion
- If the screen is wider than 768px, the Page Container will be center aligned 
```

  <PageContainerFull />
</Example>

> **Use cases may include:**

Reviewer: The use of "may" here is a subtle difference between the others that are more definitive. I am worried people will either miss the difference all together and assume that a table should use the "full" page or will wonder "When is it okay to use a table as full width"? 

We can enhance this in future, but I think we should either give some elaboration here for each bullet point such as "complex tables that would otherwise require lots of horizontal scrolling" or link to a section with the appropriate guidance.

Author: Adding this comment ☝️  to the Figma file so it doesn't get lost when this PR is merged.

- Charts 
- Maps 
- Tables

For guidelines and templates for Page Content, see [Content Layout](/templates/content-layout).

<Link href="/templates/content-layout" passHref>
  <Button 
    primary 

>     label="See Content Layout Next" 

Reviewer: ```suggestion
    label="See Content Layout" 
```
OR
```suggestion
    label="See Content Layout Guidance" 
```
